tests/main/views/test_brief_response.py

```diff
@@ -866,3 +869,16 @@ def test_filter_brief_response_list_for_all_statuses(self):
 
         assert res.status_code == 200
         assert len(data['briefResponses']) == 4
+
+    def test_filter_responses_awarded_yesterday(self):
```

Comment:
I'd rename the test to `test_filter_brief_responses_awarded_yesterday` for consistency with the others.